### DISCUSSION TITLE
Fabrikt GitHub organisation

### DISCUSSION THREAD

**Discussion by ulrikandersen** [cjbooms/fabrikt#329] [Ideas]
To foster a stronger community around Fabrikt—which I believe is by far the best OpenAPI Kotlin code generator—I think it would be valuable to set up a GitHub organization. This organization could serve as a home for Fabrikt as well as any auxiliary tools that might be created in the future.

Examples of such tools could include the existing Gradle plugin, potential GitHub Actions, or even an interactive playground 💡.

I’d love to hear your thoughts on this, @cjbooms! I noticed https://github.com/fabrikt exists, so perhaps this idea has already been on your radar?

**Comment by cjbooms** [comment]
Sorry I missed this. Yes, I am fine with doing this, sounds like a good idea. However, I do not know who owns that org, it is not me

**Reply by cjbooms** [reply]
@averabaq I don't suppose it was you who created the Fabrikt org??? 🤞

**Reply by averabaq** [reply]
Nope, it wasn't me either 😞 The org appears to be empty, maybe we could try to contact the owner and ask for the transfer to us. Or just pick another org name 🤷

**Comment by cjbooms** [comment]
Happy to say that this is done, and I have validated that publishing and builds work just fine.
I have also bought the http://fabrikt.io domain, so I think at some point we should also republish fabrikt under this new group-Id.

@acanda there is space in the org for you gradle plugin. If you want to move it here I'm happy to grant you admin access. But not sweat if you don't

**Reply by acanda** [reply]
Thank you for the offer. It might make sense to move the gradle plugin to the org in the future, e.g. when I have to change the major version of the plugin. Then I could also publish the plugin under the same group id. As long as the plugin stays on 1.x, I don't want to publish it under a new id though and will keep it in the current repository.
